doc/source/metadata/Probabilities.rst

```diff
@@ -0,0 +1,346 @@
+    ``mean, mean_absolute_value``,
+    ``mean_of_upper_decile``,
+    ``mode``,
+    ``range``,
+    ``root_mean_square``,
+    ``standard_derivation``,
+    ``sum, sum_of_squares``,
+    ``variance``.
+
+If any method other than ``point`` is specified for a given axis,
+then bounds should also be provided for that axis.
+(``point`` is essentially no statistical processing applied on that axis,
+and is unusally omitted, but could be included if it improver clarity.)
+For example, a one-dimensional array of maximum air temperatures,
+could be represented as:
+
+.. code-block:: python
+
+    variables: 
+        float air_temperature(time) ;
+            air_temperature:cell_methods = "time: maximum" ;
+        int64 time ;
+            time:bounds = "time_bnds" ;
+            time:units = "seconds since 1970-01-01 00:00:00" ;
+            time:standard_name = "time" ;
+            time:calendar = "gregorian" ;
+	int64 time_bnds(bnds) ;
+
+The ``time_bnds`` values define the periods over which 
+the statistical processing has been calculated. 
+If more than one cell method is to be indicated,
+they should be arranged in the order they were applied.
+The left-most operation is assumed to have been applied first. 
+
+It is possible to indicate more precisely how the cell method was applied
+by including extra information in parentheses after the method.
+This information includes standardized and non-standardized parts.
+The only standardized option is ``interval``,
+used to provide the typical interval between the original data values
+to which the method was applied,
+in the situation where the present data values are statistically representative 
+of original data values which had a finer spacing.
+
+.. warning::
+
+    It is important to understand that the following example, 
+    does not (necessarily) represent a 1-hour maximum temperature,
+    as the period over which the maximum is derived is included in the bounds
+    (not shown), but rather that the temperature data used to calculate
+    the maximum was provided in 1-hour steps.
+
+        .. code-block:: python
+
+            air_temperature: cell_methods = "time: maximum (interval: 1 hour)" ;
+
+The non-standardised part is included in the same way,
+unless there is also a standardised part,
+in which case it is preceded by a ``comment:`` statement.
+For example:
+
+.. code-block:: python
+        
+    air_temperature: cell_methods = "time: mean (time-weighted)" ;
+ 
+Cell methods can be used to describe any simple statistical method
+that has been apply to a variable.
+However, this leads to the question over over whether a statistical process
+that has been applied is significant for, or of relevance to, the end user?
+This is likely to depend on exactly who that user is
+and what they need to know about the data.
+Two extreme examples of how the cell methods might be presented
+for the same varaible (in this case a maximum temperature in a period)
```

Comment:
```suggestion
for the same variable (in this case a maximum temperature in a period)
```

Reply:
Think I need the spell checker plugin. Thanks.